pyanaconda/payload/rpmostreepayload.py

```diff
@@ -247,12 +243,7 @@ def _install(self, data):
                                    Variant('a{sv}', pull_opts),
                                    progress, cancellable)
         except GError as e:
-            exn = PayloadInstallError("Failed to pull from repository: %s" % e)
-            log.error(str(exn))
-            if errors.errorHandler.cb(exn) == errors.ERROR_RAISE:
-                progressQ.send_quit(1)
-                util.ipmi_abort(scripts=self.data.scripts)
```

Comment:
Is dropping the [IPMI](https://en.wikipedia.org/wiki/Intelligent_Platform_Management_Interface) call intentional here (eq. the task will do the IPMI calls by itself) ?

Reply:
If I'm not mistaken, IPMI calls should definitely stay there.

Reply:
Fix in progress...

Reply:
Thanks! :)

Reply:
I have fixed some places where the IPMI events where not reported before the exit, but in general, the exit handling is a mess. I have created a new issue and described some of the issues, but the fixes are out of scope of this PR.